src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -3205,8 +3205,10 @@ private static void NativeCompletionProcessCommands(string wordToComplete, strin
                     if (pattern.IsMatch(completionText))
                     {
                         var listItemText = completionText;
```

Comment:
There is some weird case here there I have a null/empty process name on macOS. Is that expected or a bug somewhere in the process code?